lib/preprocess.js

```diff
@@ -89,21 +92,33 @@ export const preprocess = async ({
   })
 }
 
-export const fetchMeasurementsViaClient = async (web3Storage, cid) => {
-  const res = await web3Storage.get(cid)
-  const files = await res.files()
-  const measurements = JSON.parse(await files[0].text())
-  return measurements
-}
-
-export const fetchMeasurementsViaGateway = async (cid) => {
-  const res = await fetch(`https://${encodeURIComponent(cid)}.ipfs.w3s.link/measurements.json`)
+export const fetchMeasurementsTrustless = async cid => {
```

Comment:
Now that we have only one implementation for fetching measurements, I propose to rename the function back to `fetchMeasurements`.